src/styles/antd-overrides/steps.scss

```diff
@@ -5,6 +5,22 @@
   line-height: 24px;
 }
 
+.ant-steps
+  .ant-steps-item:not(.ant-steps-item-active)
+  > .ant-steps-item-container[role='button']:hover
+  .ant-steps-item-title,
+.ant-steps
+  .ant-steps-item:not(.ant-steps-item-active)
+  > .ant-steps-item-container[role='button']:hover
+  .ant-steps-item-subtitle,
+.ant-steps
+  .ant-steps-item:not(.ant-steps-item-active)
+  > .ant-steps-item-container[role='button']:hover
+  .ant-steps-item-description {
+  color: var(--text-primary) ;
+}
```

Comment:
**suggestion** i think we can use scss `&` selector to reduce the code here: (I haven't tested this so please check!)

```suggestion
.ant-steps
.ant-steps-item:not(.ant-steps-item-active)
> .ant-steps-item-container[role='button']:hover {
  &.ant-steps-item-title,
  &.ant-steps-item-subtitle,
  &.ant-steps-item-description {
    color: var(--text-primary) ;
  }
}
```

Reply:
Yup, it is better to use scss here as we are using it in the file, thanks for the suggestion!

I used your code but tweaked it a bit! 🍻